cli/cmd/connect.go

```diff
@@ -53,7 +53,8 @@ func NewConnectCmd() *cobra.Command {
 			"  COMMAND | tt connect (<APP_NAME> | <APP_NAME:INSTANCE_NAME> | <URI>)" +
 			" [flags]\n" +
 			"  COMMAND | tt connect (<APP_NAME> | <APP_NAME:INSTANCE_NAME> | <URI>)" +
-			" [flags] [-f-] [-- ARGS]\n\n" +
+			" [flags] [-f-] [-- ARGS]\n" +
+			"  \\shortcuts - to get full list of available shortcuts in go prompt\n\n" +
```

Comment:
get the full list of available hotkeys for go prompt

Reply:
fixed